react-app/src/App.js

```diff
@@ -0,0 +1,697 @@
+import React from 'react';
+
+class App extends React.Component {
+	constructor(){
+		super();
+		this.state={ 
+			currentEvent :'====',
+			currentKeyCodes : []
+		}
+		this.currentKeysPressed= [];
+		this.down = this.down.bind(this);
+		this.up = this.up.bind(this);
+		this.keyCodeMaps = {
+			27: 'ESC',
```

Comment:
The quoting here looks odd to me but probably just work; though, I'd say is definitely something stylistic to consider.

Reply:
do you mean you would like " " better ?
i usually use for stings ' ' just because i like to do " " for the values inside the html/ jsx
But like you said both would work.
I think i will look into what formatter we could add to the pre commit. So that at least everything is consisted throughout  the project

Reply:
Sorry it didn't show what I was meaning very well but if you look down a bit lines (+22 to +31) suddenly the keys and values are both double quoted and then further down it goes back to the keys being unquoted and the values being single quoted. I know js likes to be a bit loose with typing but I wanted to make sure I pointed it out in case it *could* cause issues

Reply:
ok i just changed the strings so that we use ' when possible.
The only strings that will always be "incorrect"
will be '"' and "'"

Reply:
i will look into adding eslint